recipes/beamfit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "beamfit" %}
+{% set version = "1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/beamfit-{{ version }}.tar.gz
+  sha256: 6e6a023e52507ba44dd93eb21267c9484ee995d2a397f91eb6a929b5bbbd313c
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=64
+    - wheel
+    - numpy
+    - pip
+  run:
+    - python
+    - matplotlib-base
+    - scipy
+    - {{ pin_compatible('numpy') }}
```

Comment:
I believe you can just change this to `numpy` according to:
https://conda-forge.org/docs/maintainer/knowledge_base/#building-against-numpy